lib/terraforming/cli.rb

```diff
@@ -199,13 +199,16 @@ def vgw
       execute(Terraforming::Resource::VPNGateway, options)
     end
 
-
     private
 
-    def execute(klass, options)
+    def configure_aws(options)
```

Comment:
__[method_definition_validator]__
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L14](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L14)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L19](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L19)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L24](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L24)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L29](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L29)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L34](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L34)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L39](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L39)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L44](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L44)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L49](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L49)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L54](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L54)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L59](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L59)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L64](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L64)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L69](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L69)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L74](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L74)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L79](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L79)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L84](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L84)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L89](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L89)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L94](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L94)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L99](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L99)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L104](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L104)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L109](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L109)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L114](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L114)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L119](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L119)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L124](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L124)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L129](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L129)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L134](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L134)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L139](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L139)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L144](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L144)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L149](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L149)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L154](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L154)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L159](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L159)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L164](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L164)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L169](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L169)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L174](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L174)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L179](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L179)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L184](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L184)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L189](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L189)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L194](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L194)
* execute is undefined. Did you mean? configure_aws at [lib/terraforming/cli.rb#L199](/dtan4/terraforming/blob/7105c81c115307fc071af7ccc26e39c58ca79c82/lib/terraforming/cli.rb#L199)

<!-- Comment by [SideCI](https://sideci.com) -->